demos/python_demos/bert_question_answering_embedding_demo/bert_question_answering_embedding_demo.py

```diff
@@ -0,0 +1,338 @@
+#!/usr/bin/env python3
```

Comment:
Please set linux execution permissions for the file.

Reply:
ok